dockstore-webservice/src/main/java/io/dockstore/webservice/core/Container.java

```diff
@@ -221,8 +221,8 @@ public String getPath() {
             StringBuilder builder = new StringBuilder();
             if (registry == Registry.QUAY_IO) {
                 builder.append("quay.io/");
-            } else {
-                builder.append("registry.hub.docker.com/");
+                // } else {
```

Comment:
It was previously, but this leads to ambiguity if both the registry and toolname are optional. i.e. `<registry>/organization/image/<toolname>` is problematic and breaks some some code we have.
 
i.e. Does `funky1/funky2/funky3` refer to an organization called funky2 on a registry called funky1 with no toolname? Or does it refer to an image called funky2 stored on Docker Hub with a toolname of funky3?